packages/apps-config/src/api/params/proposalThresholds.ts

```diff
@@ -1,38 +1,31 @@
 // Copyright 2017-2021 @polkadot/app-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { ApiPromise } from '@polkadot/api';
-
 import { KULUPU_GENESIS, KUSAMA_GENESIS, POLKADOT_GENESIS } from '../constants';
 
-const PROPOSE_THRESHOLDS: Record<string, number> = {
+export const PROPOSE_THRESHOLDS: Record<string, number> = {
   [KULUPU_GENESIS]: 1,
   [KUSAMA_GENESIS]: 0.5,
   [POLKADOT_GENESIS]: 0.6,
   default: 0.5
 };
 
-const SLASH_THRESHOLDS: Record<string, number> = {
+export const REJECT_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 0.5,
```

Comment:
Since the approve was 1, the reject probably is as well, will check.

Reply:
No, 50% is correct - https://github.com/kulupu/kulupu/blob/master/runtime/src/lib.rs#L606